specification/playwrighttesting/PlaywrightTesting/tspconfig.yaml

```diff
@@ -14,3 +14,32 @@ options:
     examples-directory: "{project-root}/examples"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/playwrighttesting.json"
     omit-unreachable-types: true
+  "@azure-tools/typespec-python":
+    package-dir: "azure-developer-microsoftplaywrighttesting"
+    package-name: "{package-dir}"
+    # emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+    package-mode: azure-dataplane
+    flavor: azure
+  "@azure-tools/typespec-ts":
+    package-dir: "microsoft-playwright-testing-rest"
+    title: Microsoft Playwright Testing
+    description: Microsoft Playwright Testing Client
+    generateMetadata: false
+    generateTest: false
+    packageDetails:
+      name: "@azure-rest/microsoft-playwright-testing"
```

Comment:
Have we agreed with this namespace? A little wired with `microsoft` prefix.

Reply:
The agreed name by architects is`@azure/microsoft-playright-testing` and for .NET is `Azure.Developer.MicrosoftPlaywrightTesting.NUnit`

Reply:
I forwarded the email